engine/gamesys/src/gamesys/gamesys_script.cpp

```diff
@@ -87,18 +139,7 @@ namespace dmGameSystem
     }
 
     dmGameObject::HInstance CheckGoInstance(lua_State* L) {
-        dmGameObject::HInstance instance = dmGameObject::GetInstanceFromLua(L);
-        if (instance == 0) {
-            dmGui::HScene scene = dmGui::GetSceneFromLua(L);
-            if (scene != 0) {
-                instance = (dmGameObject::HInstance)GuiGetUserDataCallback(scene);
-            }
-        }
-        // No instance for render scripts, ignored
-        if (instance == 0) {
-            luaL_error(L, "no instance could be found in the current script environment");
-        }
-        return instance;
+        return dmScript::CheckGOInstance(L);
```

Comment:
I didn't want to update all the current users of this (albeit internal) function.